zbuf/warning.go

```diff
@@ -6,21 +6,26 @@ import (
 	"github.com/brimsec/zq/zng"
 )
 
+type Warner interface {
+	Warn(msg string) error
```

Comment:
Consider removing the error return value, as it's not used.

```suggestion
	Warn(msg string)
```

Reply:
It's needed so that a driver.Driver implements the interface.